src/apps/seo/src/views/RedirectsManager/RedirectsTable/RedirectCreator/RedirectCreator.js

```diff
@@ -103,6 +108,14 @@ export function RedirectCreator(props) {
           />
         )}
       </span>
+
+      <span className={styles.RedirectCreatorCell}>
```

Comment:
Why did this order change?

Reply:
To follow your directions of MUI design convention of Primary button on the right hand side.
https://material.io/components/dialogs#actions

Reply:
This rule is specific to dialogs and modals. In app it will be a case-by-case dependent on layout and UX.